src/modules/logger/logger.cpp

```diff
@@ -536,57 +536,61 @@ void Logger::add_common_topics()
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
 	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("airspeed", 200);
+	add_topic("arm_auth_ack");
+	add_topic("arm_auth_request");
 	add_topic("att_pos_mocap", 50);
-	add_topic("battery_status", 300);
+	add_topic("battery_status", 500);
 	add_topic("camera_capture");
 	add_topic("camera_trigger");
-	add_topic("commander_state", 100);
-	add_topic("control_state", 100);
+	add_topic("commander_state", 200);
 	add_topic("cpuload");
-	add_topic("debug_key_value");
-	add_topic("debug_value");
-	add_topic("debug_vect");
-	add_topic("differential_pressure", 50);
-	add_topic("distance_sensor");
-	add_topic("ekf2_innovations", 50);
+	add_topic("distance_sensor", 100);
+	add_topic("ekf2_innovations", 200);
 	add_topic("esc_status", 250);
-	add_topic("estimator_status", 200); //this one is large
-	add_topic("gps_dump"); //this will only be published if gps_dump_comm is set
-	add_topic("input_rc", 100);
+	add_topic("estimator_status", 200);
+	add_topic("input_rc", 200);
 	add_topic("optical_flow", 50);
 	add_topic("position_setpoint_triplet", 200);
-	add_topic("rc_channels", 100);
-	add_topic("satellite_info");
+	add_topic("rc_channels", 200);
 	add_topic("sensor_combined", 100);
-	add_topic("sensor_preflight", 50);
-	add_topic("system_power", 300);
-	add_topic("task_stack_info");
-	add_topic("tecs_status", 20);
+	add_topic("sensor_preflight", 200);
+	add_topic("system_power", 500);
+	add_topic("tecs_status", 200);
 	add_topic("telemetry_status");
 	add_topic("vehicle_attitude", 30);
-	add_topic("vehicle_attitude_setpoint", 30);
+	add_topic("vehicle_attitude_setpoint", 100);
```

Comment:
We should consider adding a 'high rate' profile, that logs these at full rate, useful for racers for instance, and analyzing corner cases during fast maneuvers.

Reply:
Yes definitely.